tagcompare/capture.py

```diff
@@ -175,7 +225,8 @@ def main():
     output.aggregate()
 
     configs = settings.DEFAULT.configs_in_comparisons()
-    __capture_tags_for_configs(cids=cids, pathbuilder=pathbuilder, configs=configs)
+    _capture_tags_for_configs(
+        cids=cids, pathbuilder=pathbuilder, configs=configs)
```

Comment:
I see. What can we do to make that more clear? Can the function be named something to suggest that a build is returned? Or can it be written in the docstring?

Or perhaps better: should the build be an object, and this should be a method on it?